projects/go-lib/src/lib/components/go-table/go-table.component.scss

```diff
@@ -19,78 +21,100 @@
 }
 
 .go-table__th {
-  background: $brand-color;
-  border-right: 1px solid $theme-light-border;
-  color: $base-light;
+  color: $theme-light-color;
   cursor: pointer;
-  font-weight: normal;
-  padding: 0.5rem 1rem;
+  font-weight: bold;
+  font-size: 0.75rem;
+  letter-spacing: 1pt;
+  padding: 1rem calc(1.25rem - 2px);
   position: relative;
   text-align: left;
+  text-transform: uppercase;
+  white-space: nowrap;
+  @include transition(all);
 
-  &:last-child {
-    border-right: 0;
+  &:hover {
+    background: $theme-light-app-bg;
   }
 }
 
-.go-table__th--sort-up::after,
-.go-table__th--sort-down::after {
-  border-left: 5px solid transparent;
-  border-right: 5px solid transparent;
-  content: '';
-  height: 0;
-  position: absolute;
-  right: 5px;
-  top: 50%;
-  transform: translateX(-50%) translateY(-50%);
-  width: 0;
-  @include transition(all);
+.go-table__th:first-of-type {
+  border-radius: $global-radius 0 0 0;
 }
 
-.go-table__th--sort-up::after {
-  border-top: 5px solid $base-light;
+.go-table__th:last-of-type {
+  border-radius: 0 $global-radius 0 0;
 }
 
-.go-table__th--sort-down::after {
-  border-bottom: 5px solid $base-light;
+.go-table__th__content {
```

Comment:
I _think_ it makes sense to do this as `.go-table__th-content` because it's still a child element of the `.go-table`? I've changed it to that for now.